### ISSUE TITLE
Old SwiftFormat for Xcode.app version 0.44.10 in newer zip archives

### ISSUE THREAD

**Issue by meyermarcel** [nicklockwood/SwiftFormat#653]
Zip archives `0.44.11`, `0.44.12` and `0.44.13` have `0.44.10` version for `SwiftFormat for Xcode.app` inside.

To reproduce the error execute the following commands with different versions:

```bash
# Set version
VERSION="0.44.9"

# Download zip archive, unpack it and print version number
wget -nc "https://github.com/nicklockwood/SwiftFormat/archive/$VERSION.zip" && unzip $VERSION.zip && plutil -p "SwiftFormat-$VERSION/EditorExtension/SwiftFormat for Xcode.app/Contents/Info.plist"  | grep CFBundleShortVersionString
```

Zip archives `0.44.9` has correct `0.44.9` version for `SwiftFormat for Xcode.app` inside.

Is this wanted because nothing changed for the `SwiftFormat for Xcode.app`?

**Comment by nicklockwood** [comment]
That definitely shouldn't be the case. Thanks for raising it.

**Comment by meyermarcel** [comment]
@nicklockwood thank you for this great project! 🙏

**Comment by nicklockwood** [comment]
@meyermarcel fixed in 0.44.14. As it turns out, it was just the version number that hadn't been updated - the actual binary was correct.
